lms/templates/verify_student/pay_and_verify.html

```diff
@@ -65,7 +65,6 @@
       data-platform-name='${platform_name}'
       data-course-key='${course_key}'
       data-course-name='${course.display_name}'
-      data-course-start-date='${course.start_datetime_text()}'
```

Comment:
looks like it's used in 2 places in https://github.com/edx/edx-platform/blob/master/lms/static/js/verify_student/pay_and_verify.js.